### PR TITLE
Feature/3

### PR DESCRIPTION
**Pull request by Jonathanwxlee**
Finished event card display

### CODE REVIEW

File: app/components/profile/EventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    >
+                        {props.eventData.title}
+                    </Text>
+                </View>
+                <View style={styles.eventDate}>
+                    <Text
+                        numberOfLines={1}
+                        style={{ fontSize: 20, textAlign: 'right' }}
+                    >
+                        {props.eventData.date.getMonth()+1}/{props.eventData.date.getDate()}
```
Comment (ch5zzy) [inline app/components/profile/EventCard.tsx]: Can you replace this with `{props.eventData.date.toLocaleDateString(undefined, {month: "numeric", day: "numeric"})}`? This should format the date string correctly for different locales.

File: app/App.tsx [hunk 2]
```diff
@@ -1,11 +1,44 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import React = require('react');
```
Comment (ch5zzy) [inline app/App.tsx]: Can you revert this file back to what it was on the `main` branch? You can move the test event data into `EventList.tsx` if you'd like to keep it for testing later.

**Review by ch5zzy — CHANGES_REQUESTED** [review]

File: app/components/profile/EventCard.tsx (hunk 1 above)
Comment (Jonathanwxlee) [reply app/components/profile/EventCard.tsx]: ok!

**Comment by ch5zzy** [comment]
Can you move the event files from `component/profile` into `component/event`?

**Review by ch5zzy — APPROVED** [review]
LGTM

**Pull request merged**